js/modules/modules.go

```diff
@@ -69,31 +65,108 @@ type HasModuleInstancePerVU interface {
 	NewModuleInstancePerVU() interface{}
 }
 
+// IsModuleV2 ... TODO better name
+type IsModuleV2 interface {
```

Comment:
I think this name is fine as it will probably be temporary until we remove `common.Bind()` in the next few weeks.

Let's just fix the comment.